mujoco_warp/_src/io.py

```diff
@@ -590,7 +589,7 @@ def tile(x):
   d.crb = wp.array(tile(mjd.crb), dtype=types.vec10, ndim=2)
   d.qM = wp.array(tile(qM), dtype=wp.float32, ndim=3)
   d.qLD = wp.array(tile(qLD), dtype=wp.float32, ndim=3)
-  d.mapM2M = wp.array(tile(mapM2M), dtype=wp.int32, ndim=3)
+  d.mapM2M = wp.array(mjd.mapM2M, dtype=wp.int32, ndim=1)
```

Comment:
Let’s put these new fields on Model.  I asked tassa@ why he did not put them there in C MuJoCo and he said it was hard for some reason but acknowledged they should be there